config/prow/plugins.yaml

```
@@ -446,7 +446,9 @@ config_updater:
   maps:
     label_sync/labels.yaml:
       name: label-config
-      namespace: test-pods
+      clusters:
+        test-infra-trusted:
+          - test-pods
     config/prow/config.yaml:
       name: config
```

I think we actually need to make this change for all these entries.
> Currently things are working because the default cluster is used which is the in-cluster context since the kubeconfig is omitted. ... I'm pretty sure mounting the kubeconfig will change what the default context points to because that will become the build cluster.

So I think this PR would currently cause the other configmaps managed here to start deploying to the build cluster instead of the service cluster. To avoid that we should probably specify the following for each:
```suggestion
    config/prow/config.yaml:
      name: config
      clusters:
        test-infra-trusted:
          - default
```
I'd prefer if we had an explicit in-cluster context, but I'm not sure how that would work with our current kubeconfig parsing logic.

---

I see... Maybe we should update the kubecofig parsing logic first?

---

> I think we actually need to make this change for all these entries.

Like that?

---

Yes, I think so.